src/dotnet/Chat.Service/Chats.cs

```diff
@@ -39,14 +39,16 @@ public Chats(IServiceProvider services) : base(services)
     }
 
     // [ComputeMethod]
-    public virtual async Task<ImmutableArray<Chat>> List(Session session, CancellationToken cancellationToken)
+    public virtual async Task<ImmutableArray<Chat>> List(
+        Session session,
+        CancellationToken cancellationToken)
     {
         var chatIds = await ChatAuthors.ListChatIds(session, cancellationToken).ConfigureAwait(false);
         var chats = await chatIds
             .Select(id => Get(session, id, cancellationToken))
             .Collect()
             .ConfigureAwait(false);
-        return chats.Where(c => c is { ChatType: ChatType.Group }).ToImmutableArray()!;
+        return chats.SkipNullItems().ToImmutableArray();
```

Comment:
Hmm, I typically use `.OfType<T>()` instead, but we can agree to use this method.

Reply:
imho, SkipNulls very clearly and simply tells what was done. It's not error prone (since OfType<T> allows you to specify wrong type). Also OfType is not only about skipping nulls, etc